src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -83,6 +83,15 @@ public void addAlias(Alias alias) {
         aliasMapping.addAlias(alias);
     }
 
+    /**
+     * Deletes an user-defined alias from the current mapping.
+     * @param aliasName The name of the alias to be deleted.
```

Comment:
Should there be a blank line between description and parameters?

Reply:
Done!